examples/fluent-bit/amazon-managed-service-for-prometheus/README.md

```diff
@@ -0,0 +1,53 @@
+## FireLens Example: Prometheus forward metrics - using the Fluent Bit image with init tag
```

Comment:
I think this needs a link in the main readme

Reply:
Added

Reply:
Also:

> Prometheus forward metrics

This confuses me and makes me think of forward input. I'd change to: forward metrics to Prometheus - using...

Reply:
Sure! Updated.